### DISCUSSION TITLE
Nested Docs Improvements

### DISCUSSION THREAD

**Discussion by yfi** [payloadcms/payload#4925] [Feature Requests & Ideas]
It would be great if the default collection list displayed parent/child relationships created with the nested docs plugin. 
Some kind of tree view or even as a custom sort where children are displayed directly under their parent item.

It would also be great if the breadcrumbs/parent fields could be placed deeper than root level. Currently it looks like the plugin looks for existing fields created with createBreadcrumbsField only at the root level of fields which means it won't find it within a group or tab resulting in a multiple fields named "breadcrumbs" error

**Comment by luochuanyuewu** [comment]
+1

**Reply by larsgaardsoe** [reply]
+1
